### PR TITLE
[tlul/dv] Add coverage exclusion file

### PR DESCRIPTION
**Pull request by weicaiyang**
Exclude following items
1. Unsupported toggle cov, e.g. a/d_param, user, some unsupported bits of
address and source
2. unreachable branches in prim_arbiter
3. scan mode related
4. line in tlul_assert, which isn't RTL
5. fifo clear related in prim_fifo_sync as it's unsupported
Signed-off-by: Weicai Yang <weicai@google.com>

### CODE REVIEW

**Comment by weicaiyang** [comment]
@eunchan 
We have reviewed these 2 items

- Unsupported toggle cov, e.g. a/d_param, user, some unsupported bits of
- address and source
- unreachable branches in prim_arbiter
- scan mode related

I just added following as assertion was updated and prim_fifo_sync was changed

- line in tlul_assert, which isn't RTL
- fifo clear related in prim_fifo_sync as it's unsupported

File: hw/ip/tlul/dv/cov/xbar_cov_excl.el [hunk 1]
```diff
@@ -0,0 +1,877 @@
...
+Block 1 "0" "assign disable_sva = (tlul_assert_ctrl === 1'b0);"
+CHECKSUM: "3373532301 1341275977"
+MODULE: prim_fifo_sync
+ANNOTATION: "[UNSUPPORTED]"
+Branch 1 "2323268504" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+ANNOTATION: "[UNSUPPORTED]"
+Branch 2 "3736627057" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+CHECKSUM: "3344740064 2208587804"
+INSTANCE: tb.dut
+ANNOTATION: "[UNSUPPORTED]"
```
Comment (sriyerg) [inline hw/ip/tlul/dv/cov/xbar_cov_excl.el]: Should these be [UNR] since it is impossible to hit these?

Comment (weicaiyang) [reply hw/ip/tlul/dv/cov/xbar_cov_excl.el]: these toggles are more like unsupported like we can drive param/user with some value but we don't support it.

File: hw/ip/tlul/dv/cov/xbar_cov_excl.el [hunk 2]
```diff
@@ -0,0 +1,877 @@
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: weicai
+// Format Version: 2
+// Date: Tue Nov  5 16:03:15 2019
+// ExclMode: default
+//==================================================
+CHECKSUM: "4068824375 1452893182"
+MODULE: prim_arbiter ( parameter N=3,DW=102 ) 
+ANNOTATION: "[UNR]"
```
[the remark this answers is not in the document]
Comment (weicaiyang) [reply hw/ip/tlul/dv/cov/xbar_cov_excl.el]: these 2 branches are UNR even FPV can't reach them.

if both arb_valid, arb_ready are 1, 1, then first `else if ` is true
so, second `else if` can never t have arb_valid, arb_ready ==1, 1
```
    end else if (arb_valid && arb_ready) begin
      // Latch only when requests accepted
      mask <= mask_next;
    end else if (arb_valid && !arb_ready) begin
      // Downstream isn't yet ready so, keep current request alive. (First come first serve)
      mask <= ppc_out;
    end
```

File: hw/ip/tlul/dv/cov/xbar_cov_excl.el (hunk 1 above)
Comment (weicaiyang) [reply hw/ip/tlul/dv/cov/xbar_cov_excl.el]: update output addresses to UNR

**Pull request merged**